### ISSUE TITLE
Chrome fails to start with latest versions Chrome 54/Chromedriver 2.24 

### ISSUE THREAD

**Issue by serioushaircut** [RobCherry/docker-chromedriver#6]
Just rebuild our image => got these versions 

```
container:/# google-chrome --version
Google Chrome 54.0.2840.59

container:/# chromedriver --version
ChromeDriver 2.24.417424 (c5c5ea873213ee72e3d0929b47482681555340c3)
```

the chromedriver service cannot start google chrome with option '--disable-setuid-sandbox' anymore. 

```
container:/# /opt/google/chrome/google-chrome.real --disable-setuid-sandbox
[335:335:1024/052420:FATAL:zygote_host_impl_linux.cc(107)] No usable sandbox! Update your kernel or see https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md for more information on developing with the SUID sandbox. If you want to live dangerously and need an immediate workaround, you can try using --no-sandbox.
Aborted
```

You can replace the option with '--no-sandbox', as it says and it works again.

**Comment by pearofducks** [comment]
Can confirm both the problem and fix.

**Comment by RobCherry** [comment]
Feel free to submit a PR, otherwise I will update after I have some free time to test the changes.

**Comment by RobCherry** [comment]
Fixed with 84d8a7e50c369a5a5a52f6d834bc701b5b03f964
